### PR TITLE
Add stream key on API

### PR DESCRIPTION
**Pull request by totoman**
I was added to the stream key on API.
It will allow the reference such as community and comment_count.

### CODE REVIEW

File: lib/niconico/live/api.rb [hunk 1]
```diff
@@ -165,6 +165,16 @@ def get_player_status(id, public_key = nil)
...
         }
 
+        stream = status.at('stream')
+        result[:stream] = {
+          id:                stream.at('id').inner_text,
+          default_community: stream.at('default_community').inner_text,
+          base_time:         stream.at('base_time').inner_text,
+          open_time:         stream.at('open_time').inner_text,
+          watch_count:       stream.at('watch_count').inner_text,
+          comment_count:     stream.at('comment_count').inner_text
```
Comment (sorah) [inline lib/niconico/live/api.rb]: to_i?

Comment (totoman) [inline lib/niconico/live/api.rb]: I'm sorry, I forgot. Added to to_i.

id -> "lv....."
default_community -> "co....."

Please go if there is a funny place.

**Comment by sorah** [comment]
Could you squash the commits?

**Comment by totoman** [comment]
It's done.
Please double check.

**Comment by sorah** [comment]
merged -- sorry for late! :pray:

**Pull request merged**